src/webpng.c

```diff
@@ -2,6 +2,10 @@
 #include "config.h"
 #endif
 
+#ifdef WIN32
+#include "getopt.h"
```

Comment:
Header is good way to declare functions, but I haven't seen headers in this library and this functionality is used only in one place.